setup.py

```diff
@@ -1,25 +1,11 @@
 import os
-import re
-from setuptools import setup, find_packages
+from setuptools import setup
 
 
 def read(f):
     return open(os.path.join(os.path.dirname(__file__), f)).read().strip()
 
 
-def read_version():
-    regexp = re.compile(r"^__version__\W*=\W*'([\d.abrc]+)'")
-    init_py = os.path.join(os.path.dirname(__file__),
-                           'aiomysql', '__init__.py')
-    with open(init_py) as f:
-        for line in f:
-            match = regexp.match(line)
-            if match is not None:
-                return match.group(1)
-        else:
-            raise RuntimeError('Cannot find version in aiomysql/__init__.py')
-
-
-setup(version=read_version(),
-      long_description='\n\n'.join((read('README.rst'), read('CHANGES.txt'))),
-      packages=find_packages(exclude=['tests', 'tests.*']))
+setup(
+    long_description='\n\n'.join((read('README.rst'), read('CHANGES.txt'))),
```

Comment:
Maybe just put `long_description = file: README.rst, CHANGES.txt` into `setup.cfg` and drop this file?

Also, this seems like a thing to do in a separate PR.

Reply:
thanks, implemented in #746